reactive/jenkins.py

```diff
@@ -174,6 +177,21 @@ def add_slaves(master):
             labels=slave["labels"] or ())
 
 
+@hook('jenkins-storage-attached')
+def attach():
+    status_set("maintenance", "Adding Jenkins storage")
+    homedir = storage_get()['location']
+    unitdata.kv().set("storage.dir", homedir)
+    set_state("reset.jenkins.storage")
```

Comment:
Yes the unitdata entry was an artifact from my previous code you are right it isn't useful anymore. I'll remove it.